feat(prometheus): added dashboards for prometheus-remote-write

tclayton-newr opened the pull request:
# Summary

## TODO fill me out please

<!-- DON'T DELETE THIS SECTION BELOW IF SUBMITTING A NEW QUICKSTART -->
## Pre merge checklist

<!-- This CHECKLIST SHOULD BE SUBMITTED FULLY COMPLETE WITH THE PR. IF NOT COMPLETE
THE PR REVIEW WILL BE DELAYED -->

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you check your dashboard image quality? -  Do they look good?
- [x] ~Did you check that your alerts actually work?~ N/A
- [x] ~Did you include an InstallPlan and Documentation reference?~ N/A
- [x] Did you check your descriptive content for voice, tone, spelling and grammar errors?
- [x] Did you attach images of your dashboards to the PR so we can see them working?

### Screenshots

Attach images of any visual changes, such as a dashboard here.
![image](https://user-images.githubusercontent.com/87031785/154778012-1b0a576a-d0c6-45ea-8d86-764198da97f8.png)
<img width="1371" alt="image" src="https://user-images.githubusercontent.com/87031785/155447259-383ae616-4b9a-435a-84b7-f555d768c7d3.png">

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/newrelic/newrelic-quickstarts?pullRequest=850) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/newrelic/newrelic-quickstarts?pullRequest=850) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/newrelic/newrelic-quickstarts?pullRequest=850) it.</sub>

jpvajda commented:
@tclayton-newr Thanks for the contribution, could you sign the CLA? You just need to do this once to contribute to our New Relic OSS repositories.

jpvajda commented on quickstarts/prometheus-remote-write/dashboards/prometheus-remote-write-monitoring-v2_30_3_dash.json:
```diff
@@ -0,0 +1,782 @@
+            ]
+          },
+          "linkedEntityGuids": [
+            "MTAxOTkxMDR8VklafERBU0hCT0FSRHwxNDY2MDg"
```
```suggestion
```

jpvajda replied:
Please change this value to null

jpvajda commented on quickstarts/prometheus-remote-write/dashboards/prometheus-remote-write-monitoring-v2_30_3_dash.json:
```diff
@@ -0,0 +1,782 @@
+            ]
+          },
+          "linkedEntityGuids": [
+            "MTA5OTcyMTl8VklafERBU0hCT0FSRHwyMDU5NTk"
```
```suggestion
```

jpvajda replied:
please change this value to null

jpvajda commented on quickstarts/prometheus-remote-write/dashboards/prometheus-remote-write-monitoring-v2_30_3_dash.json:
```diff
@@ -0,0 +1,782 @@
+            ]
+          },
+          "linkedEntityGuids": [
+            "MTAxOTkxMDR8VklafERBU0hCT0FSRHwxNDY2MDk"
```
```suggestion
```

jpvajda replied:
please change this value to null

jpvajda commented on quickstarts/prometheus-remote-write/dashboards/prometheus-remote-write-monitoring.json:
```diff
@@ -0,0 +1,695 @@
+            ]
+          },
+          "linkedEntityGuids": [
+            "MTAxOTkxMDR8VklafERBU0hCT0FSRHwxNDcxNDI"
```
```suggestion
```

jpvajda replied:
please change this value to null

jpvajda commented on quickstarts/prometheus-remote-write/dashboards/prometheus-remote-write-monitoring.json:
```diff
@@ -0,0 +1,695 @@
+            ]
+          },
+          "linkedEntityGuids": [
+            "MTAxOTkxMDR8VklafERBU0hCT0FSRHwxNDcxNDM"
```
```suggestion
```

jpvajda replied:
please change this value to null

jpvajda reviewed (changes requested):
Thanks for the contribution, I made some comments on LinkedEntityGuids that should be null.  You can use our sanatize.js script to clean the JSON up really quickly.

https://github.com/newrelic/newrelic-quickstarts#quickstart-components

tclayton-newr commented:
@jpvajda thank you for the feedback! I've updated the branch a bit and addressed any comments. 
I also ran into a small issue with the sanitizer that wasn't removing the `linkedEntityGuids` as I was expecting it should. I made a quick PR to address future dashboard.json like mine (there were some line returns). 
https://github.com/newrelic/newrelic-quickstarts/pull/861
